h/services/annotation_authority_queue.py

```diff
@@ -40,7 +40,7 @@ def publish(self, event_action: str, annotation_id: str) -> None:
         if not authority_queue_config:
             return
 
-        if event_action != "create" or not annotation.mentions:
+        if event_action not in {"create", "edit"} or not annotation.mentions:
```

Comment:
I reckon we should still explicitly list here the events we are interested in. Creating and editing is all we want for now.